test/util/test_path.cpp

```diff
@@ -159,6 +159,19 @@ bool initialize_test_path(int argc, const char* argv[])
         g_path_prefix = argv[1];
     }
 
+    Logger::Level log_level = Logger::Level::off;
+    if (const char* str = getenv("UNITTEST_LOG_LEVEL"); str && *str) {
+        if (auto custom_level = Logger::level_from_string(str)) {
+            log_level = *custom_level;
+        }
```

Comment:
What about writing the value of UNITTEST_LOG_LEVEL to stderr here, like we did in the makefile, or print an error (with the UNITTEST_LOG_LEVEL) if the value is invalid. Regardless, don't return exit the test on an invalid value.